guvnor-m2repo-editor/guvnor-m2repo-editor-client/src/main/java/org/guvnor/m2repo/client/upload/UploadFormViewImpl.java

```diff
@@ -195,8 +146,80 @@ private String getWebContext() {
         return context;
     }
 
-    protected void showUploadingBusy() {
-        // LoadingPopup.showMessage( "Uploading...");
+    public void showUploadingBusy() {
+        // TODO: "Uploading..." needs localization
+        BusyPopup.showMessage("Uploading...");
```

Comment:
@Hasys regarding the i18n; you'll need to add a new entry to [here](https://github.com/droolsjbpm/guvnor/blob/master/guvnor-m2repo-editor/guvnor-m2repo-editor-client/src/main/java/org/guvnor/m2repo/client/resources/i18n/M2RepoEditorConstants.java) and [here](https://github.com/droolsjbpm/guvnor/blob/master/guvnor-m2repo-editor/guvnor-m2repo-editor-client/src/main/resources/org/guvnor/m2repo/client/resources/i18n/M2RepoEditorConstants.properties). Translation of the "default" locale happens when we push to Zanata and pull translations back into the source code later.